subworkflows/canine_add_matched_rna.cwl

```diff
@@ -40,7 +40,7 @@ steps:
         source: input_vcf 
         valueFrom: $([self])
       format:
-        valueFrom: "%CHROM\t%POS0\t%END\n"
+        valueFrom: "%CHROM\\t%POS0\\t%END\\n"
```

Comment:
hmmm, is the double escape issue from YAML or CAVATICA?

Reply:
I'm not sure. So the old approach would get handed in to the tool as `"%CHROM\t%POS0\t%END\n"` (expected) but when the inputBinding happened it would substitute out the tabs and newline (unexpected). I didn't test it out in CWLtool.